sql/core/src/main/scala/org/apache/spark/sql/Dataset.scala

```diff
@@ -705,12 +701,10 @@ class Dataset[T] private[sql](
    *
    * @param right Right side of the join operation.
    * @param usingColumn Name of the column to join on. This column must exist on both sides.
-   *
-   * @note If you perform a self-join using this function without aliasing the input
-   * [[DataFrame]]s, you will NOT be able to reference any columns after the join, since
-   * there is no way to disambiguate which side of the join you would like to reference.
-   *
-   * @group untypedrel
+    * @note If you perform a self-join using this function without aliasing the input
+   *        [[DataFrame]]s, you will NOT be able to reference any columns after the join, since
+   *        there is no way to disambiguate which side of the join you would like to reference.
+    * @group untypedrel
```

Comment:
Can we exclude unrelated changes here? Also, `@note` has the consistent indentation across documentation. I guess it is not encouraged to change some of them inconsistently without a concrete reason.